ovos_utils/skills/__init__.py

```diff
@@ -71,6 +71,7 @@ def whitelist_skill(skill, config=None):
 
 
 def make_priority_skill(skill, config=None):
+    # TODO: Priority skills are deprecated
```

Comment:
lets deprecate this for good, add logs saying nothing happened and announcing full removal